ush/python_utils/environment.py

```diff
@@ -8,12 +8,12 @@
 
 
 def str_to_date(s):
-    """Get python datetime object from string.
+    """Gets Python datetime object from string.
 
     Args:
-        s: a string
+        s (str): a string
```

Comment:
Should the a be capitalized?

```suggestion
        s (str): A string
```